quic/api/test/QuicTransportBaseTest.cpp

```diff
@@ -3657,7 +3657,7 @@ TEST_F(
 }
 
 TEST_F(QuicTransportImplTest, ImplementationObserverCallbacksDeleted) {
-  auto noopCallback = [] {};
+  auto noopCallback = [](void*) {};
```

Comment:
I think you can do `QuicSocket *` here to avoid confusion. You can forward declare `QuicSocket` at the top of the file if needed to avoid including the header.

Reply:
I did this because i didn't think it matters much, but the possibility of confusion makes sense.